websites/B/BloomHost/presence.ts

```diff
@@ -0,0 +1,111 @@
+			presenceData.details = "Using the Panel";
+			presenceData.state = "Viewing: Audit Logs";
+		} else if (pathname.includes("server") && pathname.includes("import")) {
+			presenceData.details = "Using the Panel";
+			presenceData.state = "Using: The Server Importer ";
+		} else if (pathname.includes("server") && pathname.includes("startup")) {
+			presenceData.details = "Using the Panel";
+			presenceData.state = "Viewing: Startup";
+		} else if (pathname.includes("server") && pathname.includes("settings")) {
+			presenceData.details = "Using the Panel";
+			presenceData.state = "Viewing: Settings";
+		} else if (pathname.includes("server") && pathname.includes("subdomain")) {
+			presenceData.details = "Using the Panel";
+			presenceData.state = "Viewing: Subdomains";
+		} else if (pathname.includes("server") && pathname.includes("network")) {
+			presenceData.details = "Using the Panel";
+			presenceData.state = "Viewing: Ports & Proxies";
+		} else if (pathname.includes("server") && pathname.includes("backups")) {
+			presenceData.details = "Using the Panel";
+			presenceData.state = "Creating: Backups";
+		} else if (pathname.includes("server") && pathname.includes("users")) {
+			presenceData.details = "Using the Panel";
+			presenceData.state = "Viewing: SubUsers";
+		} else if (pathname.includes("server") && pathname.includes("schedules")) {
+			presenceData.details = "Using the Panel";
+			presenceData.state = "Viewing: Schedules";
+		} else if (pathname.includes("server") && pathname.includes("databases")) {
+			presenceData.details = "Using the Panel";
+			presenceData.state = "Viewing: Databases";
+		} else if (pathname.includes("server") && pathname.includes("minecraft")) {
+			presenceData.details = "Using the Panel";
+			presenceData.state = "Managing: Minecraft players";
+		} else if (pathname.includes("server") && pathname.includes("plugins")) {
+			presenceData.details = "Using the Panel";
+			presenceData.state = "Installing: Plugins";
+		} else if (pathname.includes("server") && pathname.includes("modpacks")) {
+			presenceData.details = "Using the Panel";
+			presenceData.state = "Installing: Modpacks";
+		} else if (pathname.includes("files")) {
+			presenceData.details = "Using the panel";
+			presenceData.state = "Editing: Files";
+		}
+	}
+	if (hostname === "status.bloom.host") {
+		presenceData.details = "Viewing the Status page";
+		presenceData.smallImageKey = "stream-solid";
+		if (pathname.includes("report")) {
+			presenceData.details = "Viewing the Status Page";
+			presenceData.state = `Viewing the status of: ${document
+				.querySelector(" div.portlet-body > div.row > div.col-xs-12 > span")
+				.textContent.replace(" (Recent History)", "")
+				.trim()}`;
+			presenceData.smallImageKey = "stream-solid";
+		}
+	}
+	if (hostname === "vps.bloom.host")
+		presenceData.details = "Using the VPS panel";
+	presenceData.smallImageKey = "server-solid";
```

Comment:
seems like you're missing curly brackets here aren't you?

Reply:
I had a bracket but the linter/auto checker was throwing errors about it so I removed it. The code still works

Reply:
It doesn't and the linter didn't do that if you did it properly, please fix this

Reply:
Yeah it does

https://github.com/PreMiD/Presences/runs/6148208616?check_suite_focus=true

Reply:
Actually also look at the latest check, it did that too when I removed the uneeded `presenceData.details = "Viewing the main website";`

Reply:
Those errors are on different lines. Please install eslint on your IDE so you can see these errors in the code itself. Also I recommend reading the errors and what they suggest

Reply:
I have eslint it didn't throw any errors

Reply:
> Also I recommend reading the errors and what they suggest

"Unnecessary" suggests it isn't needed and that i should be removed

Reply:
The errors also seem to be on the same lines

Reply:
The code is still not working as intended and all of these conversations need to be addressed.

Reply:
now I see the issue 🤦 fixed